daml-assistant/src/DA/Daml/Assistant/Install/Completion.hs

```diff
@@ -25,6 +26,12 @@ installBashCompletions options damlPath output =
     whenM (shouldInstallBashCompletions options damlPath) $
         doInstallBashCompletions damlPath output
 
+-- | Install bash completion script if we should.
```

Comment:
There is a bit of repetition here between the code for Bash and Zsh completions but given that it’s fairly simple code, I’m not sure we gain much by factoring it out.

Reply:
```suggestion
-- | Install zsh completion script if we should.
```